# OpenBLAS cblas.h is loacted at include/cblas.h and not include/openblas/cblas.h
# Aug 04th 2016 by B. Hajgato (Free University Brussels - VUB)
--- gimli/src/gimli.cpp.orig	2016-08-04 08:53:28.000000000 +0200
+++ gimli/src/gimli.cpp	2016-08-04 14:19:26.570541693 +0200
@@ -28,7 +28,7 @@
 #include <cstdlib>
 
 #if OPENBLAS_FOUND && !CONDA_BUILD
-    #include <openblas/cblas.h>
+    #include <cblas.h>
 // #else
 //     #include <cblas.h>
 #endif
